### DISCUSSION TITLE
[灌水]简真的太可爱了

### DISCUSSION THREAD

**Discussion by SKUN-C** [share121/inter-knot#4913] [灌水]
谁懂啊简真的戳我xp上了♡(*´∀｀*)人(*´∀｀*)♡
![IMG_20240815_172714](https://github.com/user-attachments/assets/ce36d99e-7197-434b-9b16-41e1765027ca)

**Comment by BuYaoHoYo** [comment]
好好好

**Comment by Wattloo** [comment]
我也喜欢

**Comment by Liruijia666** [comment]
原来你也有这个表情包

**Reply by SKUN-C** [reply]
我自己游戏里截的(›´ω`‹ )

**Comment by glidingpaper** [comment]
太可爱了

**Comment by konghuan7759** [comment]
WOW！
